src/labels/CustomLabels.labels

```diff
@@ -1617,6 +1617,22 @@
         <shortDescription>Warning displayed on Customizable Rollups page when enhanced RDs are enabled</shortDescription>
         <value>&lt;b&gt;WARNING: You have Enhanced Recurring Donations enabled. If you disable Customizable Rollups, rollups to Recurring Donations will no longer function.&lt;/b&gt;</value>
     </labels>
+    <labels>
+        <fullName>RD2_DisabledAdvancedMappingError</fullName>
+        <categories>Recurring Donation</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error for DIs with populated RD fields when Advanced Mapping is disabled.</shortDescription>
```

Comment:
Hi @sam-knox, do these error messages look ok?

Reply:
@KietTruongSFDO - Just added a couple of minor tweaks. Otherwise good to go!

Reply:
@sam-knox  updated those changes.